test/parallel/test-process-no-deprecation.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+// Flags: --expose_internals --no-warnings
+
+// The --no-warnings flag only supresses writing the warning to stderr, not the
+// emission of the corresponding event. This test file can be run without it.
+
+process.noDeprecation = true;
+
+const common = require('../common');
+const assert = require('assert');
+
+function listener(warning) {
+  common.fail('received unexpected warning');
```

Comment:
@jasnell lol, literally what I had written here at first. But @thefourtheye may a good point, if there are other warnings emitted in the future, it’s probably best to know about that, even if that might mean having to change the test file again.